AppConnectSwift/Babbage.podspec

```diff
@@ -1,5 +1,12 @@
-username = ENV['ARTIFACTORY_USERNAME'] || raise("You must set an artifactory username using the environment variable ARTIFACTORY_USERNAME.")
-password = ENV['ARTIFACTORY_PASSWORD'] || raise("You must set an artifactory password using the environment variable ARTIFACTORY_PASSWORD.")
+if File.exist?('local.yaml')
+    require 'yaml'
+    content = YAML.load(File.open('local.yaml'))
+    username = content['ARTIFACTORY_USERNAME'] || raise("You must set an artifactory username using the variable ARTIFACTORY_USERNAME.")
```

Comment:
might help to mention in the raised error that it's looking for this variable in the local.yaml file specifically?